docs/bigquery-gettingstarted.md

```diff
@@ -0,0 +1,150 @@
+    Note:  BigQuery has a [free tier](https://cloud.google.com/bigquery/pricing) that you can use to get started without enabling billing. At the time of this writing, the free tier allows 10GB of storage and 1TB of data processing per month.
+7. Navigate to the [Big Query console](https://bigquery.cloud.google.com)
+![BigQuery Console](images/07_bigquery_console.jpg)
+8. In order to add the HTTP Archive tables to your project, click on the dropdown menu next to your project name.   Then follow the example below to select "Display Project" and manually enter "httparchive"
+![Add HTTP Archive to BigQuery](images/08_adding_httparchive_to_bigquery.jpg)
+9. At this point you should see the httparchive tables in your BigQuery dashboard.   If you expand the httparchive project, you'll see folders for har, runs, scratchspace and urls. In the next section, we'll explore the structure of these tables so you can start digging in!
+![Setup Complete](images/09_setup_complete.jpg)
+
+## Understanding how the tables are structured
+So, now you have access! But what do you have access to?
+
+Most of the analysis we do is using tables in the httparchive.runs and httparchive.har folders. If you expand httparchive.runs, you'll notice that there are tables for `pages`, `requests`, `pages_mobile` and `requests_mobile`. There are sets of these tables for each HTTP Archive run dating back since 2010. And scrolling to the bottom of the folder, you'll notice that there are views that start with `latest_`. These contain the most recent data that is loaded.
+![HTTP Archive Runs Tables](images/httparchive_runs_tables.jpg)
+
+In order to understand what each of these tables contain, you can click on the table name and view the details. For example, after clicking on the 2017_08_15_pages table you can see the schema. Clicking `Details` tells you some information about the table, such as it's size and the number of rows. Clicking `Preview` shows an example of some data from the table.
+![Exploring Tables](images/exploring_data_pages_tables.jpg)
+
+
+Some of the types of tables you'll find useful when getting started are:
+### httparchive.runs Tables:
+* yyyy_mm_dd_pages:
+    * Each row contains details about a single page including timings, # of requests, types of requests and sizes.
+    * Information about the page load such # of domains, redirects, errors, https requests, etc
+    * Summary of different caching parameters
+    * CDN indicator
+    * Each page URL is associated with a "pageid"
+* yyyy_mm_dd_requests:
+    * Every single object loaded by all of the pages.
+    * Each object has a requestid and a pageid.  The pageid can be used to JOIN the _pages table.
+    * Information about the object, and how it was loaded
+    * Contains some response headers for each object
+    * These tables are very large (47GB as of Aug 2017)
+* yyyy_mm_dd_pages_mobile and yyyy_mm_dd_requests_mobile:
+    * Same as above, buf for mobile pages.
+ 
+### httparchive.har Tables:
```

Comment:
Before going into the features of the tables, could you briefly introduce what a HAR is?